apps/web/src/app/(main)/onboarding/stepper.tsx

```diff
@@ -0,0 +1,34 @@
+import { api } from '~/lib/trpc/server';
+
+import { CodeExampleStep } from './form/code-example-step';
+import { CreateFormStep } from './form/create-form-step';
+
+const Stepper = async () => {
+  const onboardingForm = await api.form.getOnboardingForm();
+  const form = onboardingForm[0]?.formId;
```

Comment:
If a user goes back to onboarding, we should make them create a new form all the time, because I broke this by deleting all my actual forms, which the onboarding form is part of, but I still get greeted with the created onboarding form during the previous onboarding.

Reply:
The best thing is to update the forms so that if you delete a form, it deletes any onboardingForms that are related to it. Then when you go there after, you can create a new form. onboarding should be once, the best place to create form is the dashboard.